tower-http/Cargo.toml

```
@@ -42,7 +42,7 @@ hyper = { version = "0.14", features = ["full"] }
 once_cell = "1"
 tokio = { version = "1", features = ["full"] }
 tower = { version = "0.4.10", features = ["buffer", "util", "retry", "make"] }
-tracing-subscriber = "0.2"
+tracing-subscriber = "0.3"
```

Updating this will also require adding the `env-filter` feature. I don't think we should update it in this PR. Lets do that in another PR.

```suggestion
tracing-subscriber = "0.2"
```